output/buffer.md

```diff
@@ -0,0 +1,114 @@
+# buffer
+
+The `buffer` output plugin buffers and re-labels events.
+This plugin is similar to [out_relabel](relabel.md), but uses [buffer](../buffer/).
+
+It is included in Fluentd's core (since v1.18.0).
+
+## Example Configuration
+
+```
+<source>
+  @type udp
+  @label @buffer
+  tag foo.udp
+  <parse>
+    @type none
+  </parse>
+</source>
+
+<label @buffer>
+  <match **>
+    @type buffer
+    @label @ROOT
+    <buffer>
+      path /path/to/buffer
+    </buffer>
+  </match>
+</label>
+
+<match foo.**>
+  @type stdout
+</match>
+```
+
+In the above example, events emitted by `in_udp` are once stored in the buffer of this plugin, then re-routed and output by `out_stdout`.
```

Comment:
emit could be used for sending outside of Fluentd.
So, we need to use ingested instead of emited in this case.
```suggestion
In the above example, events ingested by `in_udp` are once stored in the buffer of this plugin, then re-routed and output by `out_stdout`.
```